addons/repair/migrations/13.0.1.0/openupgrade_analysis.txt

```diff
@@ -1,16 +1,20 @@
 ---Models in module 'repair'---
 new model repair.tags
+# NOTHING TO DO
```

Comment:
You have to do a copy of this file, call it `openupgrade_analysis_work.txt`, and do the modifications there.